pkg/controller/backup_controller_test.go

```diff
@@ -159,6 +161,15 @@ func TestProcessBackupValidationFailures(t *testing.T) {
 		},
 	}
 
+	// Set up all of our fakes outside the test loop
```

Comment:
I would move the instantiation of this stuff inside the test loop, alongside the other instantiations. This is generally our pattern since each iteration of the test loop is its own test case and we want to ensure all of the objects are freshly instantiated for each case.

As part of that, you probably want to drop `t.Helper()`, and you can use the logger that's already being instantiated inside the loop.

This comment applies for all of the test functions in this file.